src/Input/Parser.php

```diff
@@ -247,6 +247,28 @@ protected function register(Parameter $param)
 
         $this->set($name, $param->default());
     }
+    
+    /**
+     * unRegister a new argument/option.
+     * 
+     * @param string $name
+     */
+    public function unRegister($name)
+    {
+        if (isset($this->_values[$name])) {
```

Comment:
for unset no need to check first with isset.

also multiple unset can be combined

eg: `unset($a['key1'], $a['key2']);`